includes/class-freemius.php

```diff
@@ -264,9 +269,24 @@ private function __construct( $slug ) {
 
 			$this->_plugin = FS_Plugin_Manager::instance( $this->_slug )->get();
 
+			/**
+			 * @todo Check if there is a better way to set $this->_storage->was_plugin_loaded to true for "theme" module.
+			 * If there is a better way to do that, the following if-else block will not be needed anymore and the original
+			 * code can be brought back.
+			 *
+			 * Since the current code immediately sets $this->_storage->was_plugin_loaded to "true" for "theme" module,
+			 * should_stop_execution() will return "false" and the "few-plugin-tweaks" admin notice can be added with an
+			 * empty title. So the following if-else block ensures that the admin notice will always have a title.
+			 */
+			if ( ! is_object( $this->_plugin ) && $this->is_theme() ) {
```

Comment:
@fajardoleo why not to use `$plugin_title = $this->get_plugin_name();` in both cases? (theme & plugin)